docs/DeveloperGuide.adoc

```diff
@@ -729,9 +729,83 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
```

Comment:
maybe we can change the so that i can... header to so that... :D

Reply:
Let's keep it as so that